spacy_transformers/layers/_util.py

```diff
@@ -0,0 +1,21 @@
+from thinc.api import Model, chain
+from .split_trf import split_trf_batch
+
+
+def replace_listener(model):
+    return chain(model, split_trf_batch())
+
+
+def replace_listener_cfg(tok2vec_model_cfg, listener_model_cfg):
+    result = tok2vec_model_cfg
+    if (
+        "TransformerModel" in tok2vec_model_cfg["@architectures"]
+        and "TransformerListener" in listener_model_cfg["@architectures"]
+    ):
+        result["@architectures"] = "spacy-transformers.Tok2VecTransformer.v1"
```

Comment:
I'm totally going to forget to change this string. 😬

Perhaps we could set a constant like `T2V_TRANSFORMER_ARCH_NAME` in the architectures file with a comment, `# N.B.: The architecture name is reference in `layers._util`, so if you increment the version or change to a different architecture, consider updating the `replace_listener_cfg` function.`

Reply:
I typically do a ctrl-F when updating an architecture, but sure we can have a variable name :-)

Reply:
The comment alone would be fine really, if the variable looks ugly.